### ISSUE TITLE
Distinct bus mode breaks circuit recipes on multiblocks

### ISSUE THREAD

**Issue by Y3llowL1ght** [GregTechCEu/GregTechCEu-1.19#917]
### Checked for existing issues

- [X] I have checked for existing issues, and have found none.

### Tested latest version

- [X] I have checked that this occurs on the latest version.

### GregTech CEu Version

1.1.3b-498 from github

### Recipe Viewer Installed

JEI

### Environment

single

### Cross-Mod Interaction

No

### Other Installed Mods

jei

### Expected Behavior

Multiblock recipies run fine in distinct buses mode, which allows different circuit recipies on one multi

### Actual Behavior

multiblock refuses to process stuff until distinct bus mode is toggled off on input bus filled with ingredients

### Steps to Reproduce

build working EBF -> enable distinct bus mode on input bus with circuit recipe -> it does not work -> disable distinct bus mode -> it works just fine

### Additional Information

basically found this out after upgrading community pack to latest version, tested on separate clean install with only greg and jei
(tested only ABS, EBF and wiremill on clean install)
bug affected only circuit recipies, e.g laser engraver with multiple "distinct" buses engraved stuff just fine, wiremill and cutter also worked. 
Affected multis as far as i found out were pyrolyse oven, bender, assembler, ABS and EBF

_reported this already in discord, duplicating here so it does not get lost as easily_
